packages/next-server/server/config.ts

```diff
@@ -55,7 +55,11 @@ const experimentalWarning = execOnce(() => {
 
 function assignDefaults(userConfig: { [key: string]: any }) {
   Object.keys(userConfig).forEach((key: string) => {
-    if (key === 'experimental' && userConfig[key]) {
+    if (
+      key === 'experimental' &&
+      userConfig[key] &&
+      userConfig[key] !== defaultConfig.experimental
```

Comment:
```suggestion
      userConfig[key] !== defaultConfig.experimental[key]
```

☝️ `key` is missing here, right?

Reply:
@Timer Not really :) 
It refers to https://github.com/zeit/next.js/blob/f25d4f154ac354ff3c2163a3e562f6fba960e418/packages/next-server/server/config.ts#L31

But we could have something like:
```
userConfig[key] !== defaultConfig[key]
```

Reply:
You're right!